src/test/kotlin/racingcar/domain/CarTest.kt

```diff
@@ -2,28 +2,10 @@ package racingcar.domain
 
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
-import org.junit.jupiter.params.ParameterizedTest
-import org.junit.jupiter.params.provider.MethodSource
-import java.util.stream.Stream
+import racingcar.domain.movestrategy.MoveStrategy
 
 internal class CarTest {
 
-    @ParameterizedTest(name = "{0}일 경우")
-    @MethodSource("overFour")
-    fun `이동 4보다 크거나 같을 때`(value: Number) {
-        val car = Car(Name("test"))
-        val movedCar = car.move(value)
-        assertThat(car.distance).isNotEqualTo(movedCar.distance)
-    }
-
-    @ParameterizedTest(name = "{0}일 경우")
-    @MethodSource("underFour")
-    fun `이동 숫자가 4보다 작을 때`(value: Number) {
-        val car = Car(Name("test"))
-        val movedCar = car.move(value)
-        assertThat(car.distance).isEqualTo(movedCar.distance)
-    }
-
     @Test
     fun `같은 거리인지 체크 같은 거리일 경우`() {
         val car = Car(Name("test"))
```

Comment:
생성자의 수가 늘어날수록 클래스를 사용하기 위한 유연성은 높아집니다.

```suggestion
        val car = Car("test")
```